### PR TITLE
Fixes #28975 - Add description to plugin roles

### PR DESCRIPTION
**Pull request by stejskalleos**
PR is part of  [Add default descriptions to all roles added from plugins](https://projects.theforeman.org/issues/28936) task.

### CODE REVIEW

**Comment by stejskalleos** [comment]
@adamruzicka Can you please give me some good descriptions for the roles? Thanks

File: lib/foreman_remote_execution/engine.rb [hunk 1]
```diff
@@ -102,8 +102,8 @@ class Engine < ::Rails::Engine
         ]
 
         # Add a new role called 'Remote Execution User ' if it doesn't exist
-        role 'Remote Execution User', USER_PERMISSIONS
-        role 'Remote Execution Manager', MANAGER_PERMISSIONS
+        role 'Remote Execution User', USER_PERMISSIONS, 'TODO'
+        role 'Remote Execution Manager', MANAGER_PERMISSIONS, 'TODO'
```
Comment (adamruzicka) [inline lib/foreman_remote_execution/engine.rb]: Gives the user permissions to manage job templates, manage remote execution features, cancel jobs and view audit logs.

**Comment by stejskalleos** [comment]
@adamruzicka Role descriptions updated, ready for review.

File: lib/foreman_remote_execution/engine.rb [hunk 2]
```diff
@@ -102,8 +102,8 @@ class Engine < ::Rails::Engine
         ]
 
         # Add a new role called 'Remote Execution User ' if it doesn't exist
-        role 'Remote Execution User', USER_PERMISSIONS
-        role 'Remote Execution Manager', MANAGER_PERMISSIONS
+        role 'Remote Execution User', USER_PERMISSIONS, 'Role with permissions to run remote execution jobs against hosts.'
+        role 'Remote Execution Manager', MANAGER_PERMISSIONS, 'Role with permissions to manage job templates, remote execution features, cancel jobs and view audit logs'
```
Comment (adamruzicka) [inline lib/foreman_remote_execution/engine.rb]: Nitpick: One has `.` at the end and the other one doesn't. I think it should be unified

**Comment by adamruzicka** [comment]
Thank you @stejskalleos !

**Pull request merged**